app/static/js/index.js

```diff
@@ -306,36 +316,9 @@ function loadNextPage() {
                     addRow(personContainer, 'pronouns', 'Pronouns', 'comments', person);
                     addRow(personContainer, 'title', 'Title', 'tags', person);
                     addRow(personContainer, 'year', 'Graduation Year', 'calendar', person);
-                    if (person.leave) {
-                        let row = document.createElement('div');
-                        row.classList.add('row');
-                        row.classList.add('leave');
-                        let i = document.createElement('i');
-                        i.className = 'fa fa-' + 'hourglass';
-                        row.appendChild(i);
-                        let readout = document.createElement('p');
-                        readout.classList.add('value');
-                        readout.classList.add('leave');
-                        readout.textContent = 'Took Leave';
-                        row.appendChild(readout);
-
-                        personContainer.appendChild(row);
-                    }
-                    if (person.eli_whitney) {
-                        let row = document.createElement('div');
-                        row.classList.add('row');
-                        row.classList.add('eli-whitney');
-                        let i = document.createElement('i');
-                        i.className = 'fa fa-' + 'history';
-                        row.appendChild(i);
-                        let readout = document.createElement('p');
-                        readout.classList.add('value');
-                        readout.classList.add('eli-whitney');
-                        readout.textContent = 'Eli Whitney Program';
-                        row.appendChild(readout);
-
-                        personContainer.appendChild(row);
-                    }
+                    addRow(personContainer, 'leave', null, 'hourglass', person)
+                    addRow(personContainer, 'eli_whitney', null, 'history', person)
+                    addRow(personContainer, 'visiting_international', null, 'globe', person)
```

Comment:
This change will cause the Leave and Eli Whitney rows to lose their coloring.

Reply:
Is the color additionally defined somewhere else outside of `list.css`? `row` will still be given the class `'row'` and `readout` will be given the class `'leave'` (for example). That then match the `.row.leave` selector in `list.css`.